src-java/kilda-persistence-api/src/main/java/org/openkilda/persistence/repositories/FlowPathRepository.java

```diff
@@ -39,9 +39,9 @@
 
     Collection<FlowPath> findByFlowGroupId(String flowGroupId);
 
-    Collection<PathId> findPathIdsByFlowGroupId(String flowGroupId);
+    Collection<FlowPath> findPathIdsByFlowIds(Set<String> flowIds);
```

Comment:
Shouldn't we rename it to findPathsByFlowIds?

Reply:
Yes, you are right. Fixed.